tcpdump.1.in

```diff
@@ -360,7 +360,8 @@ Savefiles will have the name specified by
 .B \-w
 which should include a time format as defined by
 .BR strftime (3).
-If no time format is specified, each new file will overwrite the previous.
+Naming clashes due to a too generic time format or to its absence will result in
+the overwrite of the previous file.
```

Comment:
Suggested edit:
```
If a generic time format is provided that does not allow for a unique filename to be used then the result could be an overwrite of the previous file.
```

Reply:
Well, it's not so much of a "could", but rather a "will". Every time a new period starts, a filename is generated, and any name collision causes a truncation of the previous pcap.
How about:
```
If no time format is specified, each new file will overwrite the previous.
Whenever a generated filename is not unique, tcpdump will overwrite the preexisting data; providing a time specification that is coarser than the capture period is therefore not advised.
```

Reply:
I think that looks good.  Thanks @aieri!